### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by mutesihope**
# This is the contact form 

- add a contact form for the portfolio
- footer is added
- add a URL for users to send a message

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -566,5 +566,121 @@ <h1 class="primary-text">About me</h1>
...
+      <form action="https://formspree.io/f/mgeqkqgz" method="post">
+        <input
+          type="text"
+          name="name"
+          placeholder="Full Name"
+          required
+          maxlength="30"
+        />
+        <input type="email" name="email" placeholder="Email Address" required />
+        <textarea maxlength="500" name="message" placeholder="Enter text here"></textarea>
```
Comment (ichala) [inline index.html:584]: - [ ] Please note that All input fields should have the `required` attribute to activate the validation on submission. the textarea (the message input) doesn't have a `required` attribute kindly add that 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @mutesihope , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Design ✅ 
- Form submission works ✅ 

 ## Required Changes ♻️

- [ ] Check the comments below

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @mutesihope , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS : APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters tests are green
- [ ] Descriptive commit messages
- [ ] the project is documented in a professional way.
## Optional suggestions
- [ ] instead of `Mariane`, you may consider put your `name` since the portfolio is yours


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — COMMENTED** [review]
Hi @mutesihope,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**